katip/src/Katip/Core.hs

```diff
@@ -22,7 +22,6 @@
 -- internal mechanisms that will change without notice.
 module Katip.Core where
 
--------------------------------------------------------------------------------
```

Comment:
Is there an automatic process here that's removing all of these comment sections? This is a coding style thing we do at Soostone.

Reply:
No, just me. I usually avoid doing petty stuff like this but they rankled me. Sorry about that.